mnist/dockerfile/README.md

```diff
@@ -0,0 +1,17 @@
+# Dockerfiles
+
+These two Dockerfiles are used to build ksonnect and mnist model docker images.
+
+
+## ksonnect
+
+```
+docker build -t siji/ksonnet:v0.13.0 -f Dockerfile.ksonnet .
```

Comment:
Can we push them to kubeflow-examples by adding them to our CI system
https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md#image-auto-release

Reply:
I'd like to change this in follow up PRs.

Reply:
Can you create an issue for this please?

Reply:
Created one: https://github.com/kubeflow/examples/issues/319 and will link too this README.